wger/core/models.py

```diff
@@ -207,6 +222,14 @@ class UserProfile(models.Model):
                        validators=[MinValueValidator(10), MaxValueValidator(100)])
     '''The user's age'''
 
+    birthdate = models.DateField(verbose_name=('Date of Birth'),
+                                 help_text=_('Please use the format: mm/dd/yyyy'),
```

Comment:
Sorry, I have a quick question.

Should I be using the Html5DateField from wger/utils/fields.py for the bday field in the model and the Html5FormDateField from wger/utils/widgets.py for the form field?